### PR TITLE
Lint TensorFlow: Migrate your TensorFlow 1 code to TensorFlow 2

### PR DESCRIPTION
**Pull request by 8bitmp3**
Splitting PR https://github.com/tensorflow/docs/pull/1736 into smaller digestible bits.

- Also proposing to use "2.x" instead of "2.0" since we're currently on v2.4.

Cheers @lamberta @MarkDaoust

### CODE REVIEW

**Comment by 8bitmp3** [comment]
@lamberta This is also done - applied suggestions from https://github.com/tensorflow/docs/pull/1736. The linter CI check is ❎ because we use "native" in "native TensorFlow 2.x" ([Google style guide file](https://github.com/tensorflow/docs/blob/master/tools/tensorflow_docs/tools/nblint/style/google.py)) and the link to the TF Hub (with a search term) is funky: `[TF Hub](https://tfhub.dev/s?q=slim%20tf2)`.

File: site/en/guide/migrate.ipynb [hunk 1]
```diff
@@ -112,7 +110,7 @@
       "source": [
         "## Top-level behavioral changes\n",
         "\n",
-        "If your code works in TensorFlow 2.0 using `tf.compat.v1.disable_v2_behavior()`, there are still global behavioral changes you may need to address. The major changes are:"
+        "If your code works in TensorFlow x using `tf.compat.v1.disable_v2_behavior`, there are still global behavioral changes you may need to address. The major changes are:"
```
Comment (lamberta) [inline site/en/guide/migrate.ipynb:113]:
```suggestion
        "If your code works in TensorFlow 2.x using `tf.compat.v1.disable_v2_behavior`, there are still global behavioral changes you may need to address. The major changes are:"
```

File: site/en/guide/migrate.ipynb [hunk 2]
```diff
@@ -141,10 +139,9 @@
         "id": "_Ni9zLLvwcOR"
       },
       "source": [
-        "## Make the code 2.0-native\n",
-        "\n",
+        "## Make the code for TensorFlow 2.x\n",
```
Comment (lamberta) [inline site/en/guide/migrate.ipynb:142]:
```suggestion
        "## Create code for TensorFlow 2.x\n",
```

**Review by lamberta — COMMENTED** [review]
Thanks, this looks good.
The tfhub lint check needs to be fixed but that will have to wait until after the new year

**Review by lamberta — APPROVED** [review]

**Pull request merged**